packages/graphql_codegen_config/lib/config.dart

```diff
@@ -52,6 +52,7 @@ class GraphQLCodegenConfig extends JsonSerializable {
     this.scopes = const ["**.graphql"],
     this.addTypenameExcludedPaths = const [],
     this.generatedFileHeader = "",
+    // deprecated after removing json_serializable
```

Comment:
I think we can just remove the option to avoid confusion.

Reply:
Is there any reason for the `test/assets/include_if_null_input` test anymore?

Reply:
I guess we can leave it since it proves that the options are backwards compatible?